src/main/java/com/northconcepts/datapipeline/mailchimp/ReadMailChimpContacts.java

```diff
@@ -0,0 +1,26 @@
+package com.northconcepts.datapipeline.mailchimp;
+
+import com.northconcepts.datapipeline.core.DataReader;
+import com.northconcepts.datapipeline.core.DataWriter;
+import com.northconcepts.datapipeline.core.LimitReader;
+import com.northconcepts.datapipeline.core.StreamWriter;
+import com.northconcepts.datapipeline.job.Job;
+import com.northconcepts.datapipeline.mailchimp.list.ListMemberStatus;
+
+public class ReadMailChimpContacts {
+
+    private static final String apiKey = "api-key";
+    private static final String listId = "list-id";
+    
+    public static void main(String[] args) {
+        DataReader reader = new MailChimpListMemberReader(apiKey, listId)
+                .setStatus(ListMemberStatus.subscribed);
+        
+        reader = new LimitReader(reader, 5);
+        
```

Comment:
Any reason we need the LimitReader in this example?  

If it distracts from the main purpose of the example, let's drop it.

Reply:
Done